include/nuttx/net/net.h

```diff
@@ -266,6 +266,7 @@ struct socket
   uint8_t       s_domain;    /* IP domain: PF_INET, PF_INET6, or PF_PACKET */
   uint8_t       s_type;      /* Protocol type: Only SOCK_STREAM or
                               * SOCK_DGRAM */
+  uint8_t       s_proto;     /* Socket Protocol */
```

Comment:
This should be initialized for all address families, even if there is only one option for protocol.

Reply:
I think this is fully handled via the change in `socket.c`?

Reply:
You are right.